common/src/main/java/com/wynntils/features/user/tooltips/ItemCompareFeature.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.tooltips;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.features.properties.RegisterKeyBind;
+import com.wynntils.core.keybinds.KeyHolder;
+import com.wynntils.mc.event.InventoryKeyPressEvent;
+import com.wynntils.mc.event.ItemTooltipRenderEvent;
+import com.wynntils.mc.event.ScreenClosedEvent;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.objects.CommonColors;
+import com.wynntils.wc.custom.item.GearItemStack;
+import java.util.List;
+import java.util.Optional;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+@FeatureInfo(stability = FeatureInfo.Stability.STABLE, category = "Item Tooltips")
+public class ItemCompareFeature extends UserFeature {
+    @RegisterKeyBind
+    private final KeyHolder toggleCompareModeKeybind =
+            new KeyHolder("Compare mode", GLFW.GLFW_KEY_SPACE, "Wynntils", true, () -> {});
+
+    @RegisterKeyBind
+    private final KeyHolder compareSelectKeybind =
+            new KeyHolder("Select for comparing", GLFW.GLFW_KEY_C, "Wynntils", true, () -> {});
+
+    private static GearItemStack COMPARED_ITEM = null;
```

Comment:
Since these are not constants, I suggest normal camel case naming. (I'm surprised Spotless does not complain..?)

Reply:
Also no reason to have them static, is there? I see Qodana complains about it, too.

Reply:
And also, could you not just have like `private boolean isCompareMode() { return comparedItem != null; }`, and skip the bool field?

Reply:
Clarified this.

Reply:
I think that would also help with what seems like a hole right now. It seems that COMPARED_ITEM is not always reset to null when COMPARE_MODE_ON is disabled. I wonder if the onRenderSlot cannot be tricked into rendering "ghost" items, where COMPARE_ITEM is set but compare mode is off.